scripts/asm-installer/install_asm

```diff
@@ -1597,6 +1635,38 @@ get_cluster_labels() {
   echo "${LABELS}"
 }
 
+is_cluster_registered() {
+  local RETVAL; RETVAL=0;
+  kubectl get memberships \
```

Comment:
Why are we not using the gcloud cmd for this?

Reply:
We need to know the membership id of the cluster in order to retrieve it deterministically.